interface/app/$libraryId/Explorer/FilePath/Thumb.tsx

```
@@ -67,9 +67,9 @@ export const FileThumb = forwardRef<HTMLImageElement, ThumbProps>((props, ref) =
 
 		if (thumbType === 'thumbnail')
 			if (
-				loadState.thumbnail !== 'error' &&
-				itemData.hasLocalThumbnail &&
-				itemData.thumbnailKey.length > 0
+				loadState.thumbnail !== 'error' // &&
```

Without this check disabled the node will just fallback to the placeholder instead of making a request over P2P to the other node for the file.

Obviously disabling it like i've done here is not a permanent solution.